src/main/webapp/components/workspace/filters.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from 'react'
+import Button from '@material-ui/core/Button'
+import Divider from '@material-ui/core/Divider'
+import Section from './section'
+import Typography from '@material-ui/core/Typography'
+import { defaultFilter } from '../query-builder/filter/filter-utils'
+import Box from '@material-ui/core/Box'
+import Filter from '../query-builder/filter/individual-filter'
+
+const anyTextFilter = { ...defaultFilter, value: '*' }
+
+const FilterList = ({ filters, onChange }) =>
+  filters.map((filter, i) => (
+    <Box key={i} style={{ padding: '0px 16px' }}>
+      <Filter
+        filter={filter}
+        onChange={newFilter => {
+          // Have to copy / clone array to modify
+          const newFilters = filters.slice()
+          newFilters[i] = newFilter
+          onChange(newFilters)
+        }}
+        onRemove={() => {
+          onChange(filters.filter((filter, index) => index !== i))
+        }}
+      />
+    </Box>
+  ))
+
+const Filters = ({ onApply, onChange }) => {
+  const [filters, setFilters] = useState([])
+  const [enable, setEnable] = useState(false)
+
+  useEffect(
+    () => {
+      onChange(filters)
+    },
+    [filters]
+  )
+
+  const toggleSection = () => {
+    if (enable) {
+      setEnable(false)
+      onChange([])
```

Comment:
Can move the onChange to the useEffect to avoid potential issues with the setEnable being async